docs/source/eventing/actions/label.rst

```diff
@@ -56,4 +72,11 @@ Changes the text and color of a generic label device.
    IF 1 == 1 THEN label DEVICE tempLabel AND humiLabel TO 'No information'
    IF 1 == 1 THEN label DEVICE tempLabel TO 23.5 FOR '10 SECOND'
    IF 1 == 1 THEN label DEVICE tempLabel TO 'Bell rang' AFTER '30 SECOND'
-   IF 1 == 1 THEN label DEVICE tempLabel TO None FOR '10 MINUTE' AFTER '30 SECOND'
\ No newline at end of file
+   IF 1 == 1 THEN label DEVICE tempLabel TO None FOR '10 MINUTE' AFTER '30 SECOND'
+
+.. versionadded:: 8.1.4.
```

Comment:
Can you make this `nightly` everywhere instead of `8.1.4`

Reply:
done